website/src/css/_navbar.scss

```diff
@@ -22,3 +22,29 @@
   padding: 5px;
   font-weight: 300;
 }
+/*===========*/
+
+/* Restyle gamburger button */
+
+.navbar__toggle {
+  width: 40px;
+  height: 40px;
+  background-color: var(--detox-color-a3);
+  border-radius: 10px;
+  justify-content: center;
+  align-items: center;
+}
+
+/* Restyle search button */
+
+@media (max-width: 769px) {
+  button.DocSearch-Button {
```

Comment:
Side note for other reviewers passing by – adding a specific CSS class was **necessary** due to a clash between styling rules from the vendor and our custom ones.